src/Engine/ProtoAssociative/CodeGen.cs

```diff
@@ -3957,7 +3957,7 @@ public override int Emit(ProtoCore.AST.Node codeBlockNode, ProtoCore.Associative
             AllocateContextGlobals();
 
             core.startPC = this.pc;
-            if (core.ExecMode == ProtoCore.DSASM.InterpreterMode.kExpressionInterpreter)
+            if (core.Options.RunMode == ProtoCore.DSASM.InterpreterMode.kExpressionInterpreter)
```

Comment:
Im still trying to determine is runmode needs its own Options. Right now, the Options passed to the runtime is the same as the Option used by the compile core.

Added this investigation to: http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-6193
Targetted for this sprint